pyoptools/gui/ipynbplotutils.py

```diff
@@ -177,7 +177,7 @@ def _repr_jpeg_(self):
 
     def _buffer_to_image(self):
         h, w, _c = self.buffer.shape
-        image = PImage.fromstring("RGBA", (w, h), self.buffer)
+        image = PImage.frombytes("RGBA", (w, h), self.buffer)
```

Comment:
It should work just fine, `fromstring` was just an alias to `frombytes` (and already deprecated).